core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -219,6 +210,14 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
 
     // setupWorkspace main logic
     val traceId = TraceId(UUID.randomUUID())
+    val cors = List(
```

Comment:
I took these values from the ticket description. In Azure, we set the max age to 0 which I may test out so that we're consistent between the two clouds

Reply:
Did you have a chance to test it out?

Reply:
I was able to test this in a BEE and verify that it worked once I had changed the allowed origins to `*` with Appsec's approval since the origin in a BEE isn't `notebooks.firecloud.org`